core/src/main/java/com/expedia/adaptivealerting/core/util/MathUtil.java

```diff
@@ -16,16 +16,32 @@
 package com.expedia.adaptivealerting.core.util;
 
 public class MathUtil {
-    
+
     /**
      * Indicates whether the distance between d1 and d2 is less than or equal to the given threshold.
      *
-     * @param d1        Value 1
-     * @param d2        Value 2
-     * @param threshold Threshold
+     * @param d1
+     *            Value 1
+     * @param d2
+     *            Value 2
+     * @param threshold
+     *            Threshold
      * @return Boolean indicating whether the two value are approximately equal
      */
     public static boolean isApproximatelyEqual(double d1, double d2, double threshold) {
         return Math.abs(d1 - d2) <= threshold;
     }
+
+    public static double getSquare(double value) {
+        return Math.pow(value, 2);
+    }
+
+    public static double getSquareRoot(double value) {
+        return Math.sqrt(value);
+    }
+
+    public static double roundToThree(double value) {
+        return (Math.round(value * 1000) / 1000);
```

Comment:
This doesn't quite round to three digits. It's still a double and due to how floating point arithmetic works, there is no guarantee that this rounds to three digits.